examples/example-audit-execution/pom.xml

```diff
@@ -0,0 +1,71 @@
+<!--
```

Comment:
This should probably be in `src/it/example` and executed by invoker (somewhere in the tree).